### PR TITLE
Add file info

### PR DESCRIPTION
**Pull request by soupette**
#### Description of what you did:

This PR adds the file info to the edit modal

### CODE REVIEW

File: packages/strapi-admin/admin/src/components/LeftMenuLinkContainer/index.js [hunk 1]
```diff
@@ -87,8 +88,8 @@ const LeftMenuLinkContainer = ({ plugins }) => {
 
   return (
     <Wrapper>
-      {menus.map(section => (
-        <MenuSection>
+      {menus.map((section, index) => (
```
Comment (HichamELBSI) [inline packages/strapi-admin/admin/src/components/LeftMenuLinkContainer/index.js]: Maybe we should add an id or use the section name for the key?

File: packages/strapi-plugin-upload/admin/src/components/UploadList/RowItem.js [hunk 2]
```diff
@@ -6,7 +6,8 @@ import CardImgWrapper from '../CardImgWrapper';
 import InfiniteLoadingIndicator from '../InfiniteLoadingIndicator';
 
 const RowItem = ({
-  file,
+  // file,
```
Comment (HichamELBSI) [inline packages/strapi-plugin-upload/admin/src/components/UploadList/RowItem.js]: Do you still need this?

File: packages/strapi-plugin-upload/admin/src/components/UploadList/index.js [hunk 3]
```diff
@@ -28,10 +28,7 @@ const UploadList = ({
       <ModalSection justifyContent="space-between">
         <TextWrapper>
           <Text fontSize="md" fontWeight="bold">
-            <FormattedMessage
-              id={getTrad(titleId)}
-              values={{ number: filesToUploadLength }}
-            />
+            <FormattedMessage id={getTrad(titleId)} values={{ number: filesToUploadLength }} />
```
Comment (HichamELBSI) [inline packages/strapi-plugin-upload/admin/src/components/UploadList/index.js]: Can you add a todo comment here? Use `<IntlText ... />` instead

File: packages/strapi-plugin-upload/admin/src/components/UploadList/RowItem.js (hunk 2 above)
Comment (soupette) [reply packages/strapi-plugin-upload/admin/src/components/UploadList/RowItem.js]: You might need it when designing the card so I left it just to let you know it's already there.

**Pull request merged**